unite_contranges.py

```diff
@@ -0,0 +1,147 @@
+#Allison Towner
+#09 October 2020
+
+#continuum_selections.py in analysis code is located at:
+#in ALMA_IMF/reduction/analysis/
+
+import numpy as np
+execfile('/orange/adamginsburg/atowner/ALMA_IMF/contranges/parse_contdotdat.py')
```

Comment:
```suggestion
import sys
sys.path.append('.')
from parse_contdotdat import parse_contdotdat
```
`execfile` is deprecated and won't work on python3, which you should get used to - CASA will move away from py2 soon.

Reply:
(also this approach doesn't rely on being run on hpg...)